### DISCUSSION TITLE
user input community container

### DISCUSSION THREAD

**Discussion by Zoey2936** [nextcloud/all-in-one#3507] [Questions] [answered]
I thought about adding a community container, but I need input of the user before the container can start, is this possible? (this input needs to be injected as env to the community container)

**Comment by szaimen** [comment, marked as answer]
Hi, this is on the follow-ups list: https://github.com/nextcloud/all-in-one/discussions/3432.

For what would you need the input?

**Reply by Zoey2936** [reply]
I need the input to provide certbot an email address, but I just found a way to skip this (`--register-unsafely-without-email`)

**Reply by szaimen** [reply]
> but I just found a way to skip this (`--register-unsafely-without-email`)

Yes indeed :)
